src/js/actions/transform.js

```diff
@@ -1080,28 +1080,27 @@ define(function (require, exports) {
 
     var beforeStartup = function () {
         _artboardTransformHandler = synchronization.debounce(function () {
-            this.dispatch(events.ui.TOGGLE_OVERLAYS, { enabled: true });
-            
             // After each action we call, document model changes
             // so we re-get it
             var appStore = this.flux.store("application"),
                 nextDoc = appStore.getCurrentDocument();
 
-            return this.flux.actions.layers.getLayerOrder(nextDoc, true)
+            return this.flux.actions.layers.resetBounds(nextDoc, nextDoc.layers.allSelected)
```

Comment:
I'm surprised we want to call `resetBounds` before `resetSelection`. If we're calling the latter, that must mean we may have lost track of the selection previously. But that previous set is the set whose bounds we're resetting. Shouldn't it be the other way around?